src/global-styles.ts

```diff
@@ -215,14 +251,15 @@ export const globalStyles = css`
     margin: 0;
     border: none;
     font: inherit;
-    color: ${color.space};
+    color: ${color.foreground};
```

Comment:
Any reason why this is not `action`? Seeing that it was `space` before, but since it's a button doesn't it make sense to be blue?

Reply:
The default of a the html button is also black. Why should we change it to blue? Our default button would look like a link, which it isn't, and if you need that then you should use a link probably.

Reply:
The default of a html button has a background and border that makes it look like a button. That is not the case here, which means that if someone only uses the button element (without any classes) it wouldn't look like something clickable.

¯\_(ツ)_/¯

Reply:
We mostly use it for wrapping an icon. An info icon that opens a dialog for example. These are mostly black, so we can change this to be blue but than we need to keep these things in mind 🤔 
It is not a bad idea to force you to make clickable things a `action` color, but this would change the designs ofcourse

Reply:
Ah makes sense. Wouldn't do any unnecessary changes like that here since this project is big enough, maybe we can create a new Github Issue to look into this at some point instead.